src/App.vue

```diff
@@ -1,12 +1,31 @@
 <template>
   <div id="app">
-    Hello, Amanda here!
+    <SearchBar @termChange="onTermChange"></SearchBar>
   </div>
 </template>
```

Comment:
You could divide HTML, CSS and JS in different files, what do you think ?

Reply:
So... I should create a folder for each component and inside it .js, .css and .vue with the template tag?

Reply:
Yeah, you could create a folder for each component and create .js, .css , .html and .vue.

Inside vue component , you import whole files. 

You can look in portal zap, we can make this form

Reply:
Done it.